docs/advanced.rst

```diff
@@ -348,3 +348,21 @@ If you want to save a model without a historical record, you can use the followi
 
     poll = Poll(question='something')
     poll.save_without_historical_record()
+
+History Diffing
+-------------------
+
+When you have two instances of the same HistoricalRecord (such as the HistoricalPoll example above),
+you can perform diffs to see what changed. This will result in a ModelDelta containing properties including
```

Comment:
This will result in a `ModelDelta` with the following properties:
1. A of changes between each of the two historical records
2. A list with the names of all fields that differed from one record to the other
3. The old and new records